components/motor/gpio/encoded.go

```diff
@@ -172,7 +173,14 @@ func (m *EncodedMotor) makeAdjustments(ctx context.Context, goalRPM, goalPos, di
 			currentRPM = 0
 		} else {
 			currentRPM = deltaPos / deltaTime
+			if currentRPM == 0 {
+				zeroRPMTracker++
+			}
+		}
+		if zeroRPMTracker > 10 {
+			m.logger.Errorf("error running motor %v, desired rpm is too low for stable motion: %v", m.Name().Name, goalRPM)
```

Comment:
I'm a bit confused what's happening here... doesn't the issue occur at low non-zero speeds? so this would only detect if the speed is actually 0

Reply:
yeah this only triggers if you set the motor speed to something thats too slow to properly spin the motors. at low speeds I saw the motors will actually say 0 rpm, but there isn't really anything else we can do

Reply:
okay that makes sense